python/pyarrow/_parquet.pyx

```diff
@@ -319,6 +319,84 @@ cdef _box_flba(ParquetFLBA val, uint32_t len):
     return cp.PyBytes_FromStringAndSize(<char*> val.ptr, <Py_ssize_t> len)
 
 
+cdef class GeoStatistics(_Weakrefable):
+    """Statistics for columns with geospatial data types"""
+
+    def __cinit__(self):
+        pass
```

Comment:
We should prevent the user from calling `GeoStatistics()` because the returned object would be crashy. There are similar patterns elsewhere in the codebase.